douglas.md

```diff
@@ -0,0 +1,122 @@
+# Third party cookies are phasing out, now what?
+
+## Introduction
+- Third party cookies are being phased out by browsers
+- This is a good thing for privacy
+- But it's a problem for many businesses
+- This is a brief overview of what's happening and what you can do about it
+
+Third party cookies received an ultimato. Browsers are currently phasing them out, that means they'll won't be allowed anymore if you use a browser that covers that specification, like Safari, Firefox or Chrome. This decision is not really recent, but since the deadline is near, it's time to undertstand what's up with these changes and how do they affect us. Despite this being a good thing for user's privacy, it is something that may hit hard some businesses. This is a brief overview of what's happening and what you can do about it.
```

Comment:
Eu acho que aqui é `ultimatum`

Reply:
`if you use a browser that covers that specification` -> `if you use a browser that follows that specification` ou `adheres to`